Gruntfile.js

```diff
@@ -373,7 +373,7 @@ module.exports = function(grunt) {
 		// repeat these steps when needed
 		watch: {
 			js: {
-				tasks: concatAndUglifyJs('dev'),
+				tasks: ['concurrent:dev_generate_js'].concat(concatAndUglifyJs('dev')),
```

Comment:
I used `[].concat('all', 'the', 'tasks')` so they align better, but this works just the same